tests/lib/rules/prefer-find-by.test.ts

```diff
@@ -1,18 +1,34 @@
-import { InvalidTestCase } from '@typescript-eslint/experimental-utils/dist/ts-eslint'
+import { InvalidTestCase, ValidTestCase } from '@typescript-eslint/experimental-utils/dist/ts-eslint'
 import { createRuleTester } from '../test-utils';
 import { ASYNC_QUERIES_COMBINATIONS, SYNC_QUERIES_COMBINATIONS } from '../../../lib/utils';
-import rule, { WAIT_METHODS, RULE_NAME } from '../../../lib/rules/prefer-find-by';
+import rule, { WAIT_METHODS, RULE_NAME, getFindByQueryVariant, MessageIds } from '../../../lib/rules/prefer-find-by';
 
 const ruleTester = createRuleTester({
   ecmaFeatures: {
     jsx: true,
   },
 });
 
+function buildFindByMethod(queryMethod: string) {
+  return `${getFindByQueryVariant(queryMethod)}${queryMethod.split('By')[1]}`
```

Comment:
I'm not a fan of doing this kind of "importing the function calculating something to check the same in the tests". This can lead to silent errors, so I prefer to hardcode or reimplement this kind of behaviors in testing side. Anyway, I think for this case it's fine.

Reply:
I generally, agree, but I was using that function everywhere and it was so small... I couldn't resist the temptation (?)